docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -1,6 +1,6 @@
 ﻿.. _whatsnew_0700:
 
-v0.7.0 (MONTH DAY, YEAR)
+v0.7.0 (December 18, 2019)
 ------------------------
```

Comment:
I think sphinx will emit a warning if the `--` is not the same length as the line above.

Reply:
Fixed. Thanks for the warning!